News: Our Zulip Chat Exceeds 3,000 Members (#411)
mixxxdj/website: content/news/XXXX-XX-XX-zulip-3000_members.md

- Cluttered threading and confusing message flow  
- No shared cloud storage for files and images  
- No video chat

As the project and number of contributors grew, we needed a platform that could keep conversations organized and accessible to everyone.

Zulip, which was open-sourced in 2015, was very promising and convinced us after a short evaluation period.
It's unique thread system was the ideal solution to replace structured email threads and instant chat rooms.

Whenever a topic drifts, we can split it into a new thread, so conversations stay focused and are easy to follow.  
This approach makes it possible to collaborate asynchronously across time zones without missing important context.
No need to hang around in chat rooms for not missing an important conversations.

Thanks to Zulip, it’s now much easier to stay up to date. Our team collaboration and community engagement have both
improved significantly. Zulip works particularly well for open-source projects where many topics run in parallel, and where structure and focus are key to success.

> It can be used with a web frontend of a Mobile app for Android and IOs.

Reviewer: We should shout out the new mobile app they have release a few month ago, with the full Flutter rewrite IMHO. The new experience is really good!

title: Our Zulip Chat Exceeds 3,000 Members
authors: Daniel Schürmann
status: draft
tags: zulip
comments: yes

> #### Our Zulip Chat Exceeds 3,000 Members

Reviewer 2: ```suggestion
#### Mixxx Zulip Chat Exceeds 3,000 Members
```

title: Our Zulip Chat Exceeds 3,000 Members
authors: Daniel Schürmann
status: draft
tags: zulip
comments: yes

#### Mixxx Zulip Chat Exceeds 3,000 Members

> Since moving our community discussions from our mailing list to **Zulip** in 2018, we have never looked back.  

Reviewer 3: Consider linking to the https://zulip.com/ website for folks who aren't familiar with Zulip?

- Cluttered threading and confusing message flow  
- No shared cloud storage for files and images  
- No video chat

As the project and number of contributors grew, we needed a platform that could keep conversations organized and accessible to everyone.

Zulip, which was open-sourced in 2015, was very promising and convinced us after a short evaluation period.
Its unique thread system was the ideal solution to replace structured email threads and instant chat rooms.

Whenever a topic drifts, we can split it into a new thread, so conversations stay focused and are easy to follow.  
This approach makes it possible to collaborate asynchronously across time zones without missing important context.
No need to hang around in chat rooms to avoid missing an important conversation.

Thanks to Zulip, it’s now much easier to stay up to date. Our team collaboration and community engagement have both
improved significantly. Zulip works particularly well for open-source projects where many topics run in parallel, and where structure and focus are key to success.

> It can be used with a web frontend or a mobile app for Android and iOS.

Reviewer 3: ```suggestion
```

> In addition to the web interface, Zulip has also a nice mobile app forAndroid or iOS.

Reviewer 3: ```suggestion
In addition to the web interface, Zulip has also a nice mobile app for Android or iOS.
```

> It has recently received mayor update, a full Flutter rewrite with fast natively compiled code.

Reviewer 3: ```suggestion
It has recently received a major update, a full Flutter rewrite with fast natively compiled code.
```

> This allows to stay in touch easily while commuting or when away from a desktop.

Reviewer 3: ```suggestion
This allows us to stay in touch easily while commuting or when away from a desktop.
```

> We enjoy unique feature like the time widget which is always showing local time, the poll widget for fast decision making and the pronouns fieldthe pronouns fieldthat helps people address others correctly when the gender associated with their name isn’t clear.

Reviewer 3: ```suggestion
We enjoy unique features, like the time widget, which is always showing local time, the poll widget for fast decision making, and the pronouns field that helps people address others correctly when the gender associated with their name isn’t clear.
```
